packages/ember/src/components/card.gts

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+
+interface CardSignature {
+  Element: HTMLDivElement;
+  Args: {
+    // eslint-disable-next-line no-unused-vars
+    onClick?: (evt: MouseEvent) => unknown;
+  };
+  Blocks: {
+    'card-header': [];
+    'card-body': [];
```

Comment:
We're already in a `Card` component, I don't think we need that in the name of the block:
```suggestion
    header: [];
    body: [];
```